process.go

```diff
@@ -142,7 +143,17 @@ type ProcessMessageHandler interface {
 //
 // This interface is currently equivalent to [any], but is a distinct type to
 // allow future extensions without breaking compatibility.
```

Comment:
The comment isn't accurate _any_more, is it?

```suggestion
// ProcessRoot is an interface for an application's working representation
// of a process instance used within [ProcessMessageHandler]
// implementations.
```

Reply:
Nice catch, ta.